src/desktop/apps/artwork/components/commercial/templates/price.jade

```diff
@@ -8,11 +8,9 @@ if artwork.sale_message || isPermanentCollection
         .artwork-commercial__price-label Price
         .artwork-commerical__price
           | #{artwork.sale_message}
-          span(
-            class= artwork.is_price_range ? 'help-tooltip' : ''
-            data-message= 'The range is an approximate indication of the work’s price point, and the exact price is quoted upon request.'
-            data-anchor= 'bottom-right'
-          )
+          if artwork.is_price_range
+            .tooltip-question-container
+              != stitch.components.TooltipQuestion({horizontalAlign: 'left', message: 'The range is an approximate indication of the work’s price point, and the exact price is quoted upon request.'})
```

Comment:
@damassi - here is where I'm using the component exported in `global_react_modules.js`. So props are passed here. Maybe I'm not using it correctly?

Reply:
This surfaced a pretty interesting edge-case that i was able to fix due to the inability to accurately test around `setTimeout` in jest. Published a new version, incoming to master

Reply:
Fixed in Force #master